api/service/stagedstreamsync/stage_bodies.go

```diff
@@ -60,6 +61,11 @@ func (b *StageBodies) Exec(ctx context.Context, firstCycle bool, invalidBlockRev
 		return nil
 	}
 
+	// shouldn't execute for epoch chain
+	if b.configs.bc.ShardID() == shard.BeaconChainShardID && !s.state.isBeaconNode {
+		return nil
```

Comment:
can you add a info message here ?

Reply:
Each stage is executed once for each sync cycle, and the sync cycles will run in a loop. Therefore, this line will continuously run, and if the chain is an epoch chain, it will log the information repeatedly within a cycle, resulting in many entries in the log file. In some cases, this could lead to thousands of logs being generated per hour.